Tests/LibWeb/WebAudio/compressor-node-channel-cound-mode.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```

Comment:
You should add a LibWeb/Text test instead of just an HTML file in this directory. Those tests are run automatically, and are compared to a specific output. Instead of enumerating all possibilities for test results, you can then simply do things like `println(node.channelCountMode)` and put the expected result in the associated `.txt` file.

Reply:
Gotcha, I will try fixing this